apps/web/playwright/booking/longTextQuestion.e2e.ts

```diff
@@ -0,0 +1,423 @@
+import { loginUser } from "../fixtures/regularBookings";
+import { test } from "../lib/fixtures";
+
+test.describe("Booking With Long Text Question and Each Other Question", () => {
+  const bookingOptions = { hasPlaceholder: true, isRequired: true };
+
+  test.beforeEach(async ({ page, users }) => {
+    await loginUser(users);
+    await page.goto("/event-types");
+  });
+
+  test("Long Text and Address required", async ({ bookingPage }) => {
+    await bookingPage.goToEventType("30 min");
+    await bookingPage.goToTab("event_advanced_tab_title");
+    await bookingPage.addQuestion("textarea", "textarea-test", "textarea test", true, "textarea test");
+    await bookingPage.addQuestion("address", "address-test", "address test", true, "address test");
+    await bookingPage.updateEventType();
+    const eventTypePage = await bookingPage.previewEventType();
+    await bookingPage.selectTimeSlot(eventTypePage);
+    await bookingPage.fillAndConfirmBooking({
+      eventTypePage,
+      placeholderText: "Please share anything that will help prepare for our meeting.",
+      question: "textarea",
+      fillText: "Test Long Text question and Address question (both required)",
+      secondQuestion: "address",
+      options: bookingOptions,
+    });
+    await bookingPage.cancelAndRescheduleBooking(eventTypePage);
```

Comment:
These tests are much more readable now, which is great but the fact that there is no assertion is bizarre. I propose renaming this function to `assertBookingCanBeCanceledAndRescheduled`.

Reply:
@gitstart-calcom  looking at this function it in fixtures, it seems like this function was supposed to be split in two ? like so:
```
await bookingPage.cancelBooking() 
await bookingPage.verifyBookingCancellation()
await bookingPage.rescheduleBooking()
await bookingPage.verifyReschedulingSuccess() 
```